app/controllers/api/api_controller.rb

```diff
@@ -17,7 +17,8 @@ class ApiController < ApplicationController
       Subjects::Selector::MissingSubjectSet,
       Subjects::Selector::MissingSubjects,                 with: :not_found
     rescue_from ActiveRecord::RecordInvalid,               with: :invalid_record
-    rescue_from Api::LiveProjectChanges,                   with: :forbidden
+    rescue_from Api::LiveProjectChanges,
+      Api::ExportDisabled,                                 with: :forbidden
```

Comment:
it just gets queued up in the backend. It's a black box process already, the feedback then will be the same as it is already. Maybe we just put some text up there during SGL saying they may be delayed?